### PR TITLE
Added materials section + upcoming events

### PR DESCRIPTION
**Pull request by MoeweX**
Added materials section for logos, presentation, etc. and upcoming events... .

http://jonathanh5.github.io/stratosphere/events/

### CODE REVIEW

**Comment by StephanEwen** [comment]
Looks good, please merge

**Comment by fhueske** [comment]
@rmetzger Do you want to add the Slides of your Bitkom Big Data Talk to the materials section?

File: project/material/index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+<p>Feel free to use the material on this website for your presentations or other events related to Stratosphere.</p>
+
+<h2 class="text-primary">Logo</h2>
+<div class="row">
+	<div class="col-md-4">
+		<img src="{{ site.baseurl }}/project/material//stratosphere_logo.pdf" class"img-rounded">
+	</div>
+	<div class="col-md-3">
+		<button type="button" class="btn btn-info btn-lg btn-block" onclick="location='deinedatei.exe'">Download PDF</button>
```
Comment (rmetzger) [inline project/material/index.html:16]: Oh shit. We merged that into the live page. "deinedatei.exe"

Comment (rmetzger) [inline project/material/index.html:16]: I fixed it in commit 4a0b653d169ba8e46763b50a3b7c331b205689d7

**Pull request merged**